X Server needs to open /dev/tty1 RW on startup. Right now TTYs are
set to group write only. When X server will be started as xorg user
which belongs to tty group, we need tty1 to be group RW. We also
set tty2 because in some test scenarios X server can run on it.

diff -urN udev-208-org/rules/50-udev-default.rules udev-208/rules/50-udev-default.rules
--- udev-208-org/rules/50-udev-default.rules	2010-12-10 04:23:54.000000000 -0800
+++ udev-208/rules/50-udev-default.rules	2013-10-02 18:19:25.451793617 -0700
@@ -15,6 +15,7 @@
 SUBSYSTEM=="tty", KERNEL=="ptmx", GROUP="tty", MODE="0666"
 SUBSYSTEM=="tty", KERNEL=="tty", GROUP="tty", MODE="0666"
 SUBSYSTEM=="tty", KERNEL=="tty[0-9]*", GROUP="tty", MODE="0620"
+SUBSYSTEM=="tty", KERNEL=="tty[1-2]", GROUP="tty", MODE="0660"
 SUBSYSTEM=="vc", KERNEL=="vcs*|vcsa*", GROUP="tty"
 KERNEL=="tty[A-Z]*[0-9]|pppox[0-9]*|ircomm[0-9]*|noz[0-9]*|rfcomm[0-9]*", GROUP="uucp"
 
